go.mod

```diff
@@ -1,8 +1,8 @@
 module github.com/nitishm/go-rejson
 
+go 1.14
+
 require (
-	github.com/go-redis/redis v6.15.1+incompatible
-	github.com/gomodule/redigo v2.0.0+incompatible
-	github.com/onsi/ginkgo v1.8.0 // indirect
-	github.com/onsi/gomega v1.5.0 // indirect
+	github.com/go-redis/redis/v7 v7.4.0
+	github.com/gomodule/redigo v1.8.2
```

Comment:
In addition to `go-redis` version, I have also upgraded `redigo` version. As I personally used `go-rejson` with the upgraded version of `redigo` and it has no difference.

Reply:
LGTM